### PR TITLE
Update RTD build OS to Ubuntu 22.04

### PR DESCRIPTION
**Pull request by eddddddy**
Fix failures in doc builds

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -32,6 +32,7 @@ torchvision==0.12.0; sys_platform == "darwin" and python_version == "3.10"
 jinja2==3.0.3
 rustworkx==0.12.1
 networkx==2.6
+urllib3<2
```
Comment (timmysilv) [inline doc/requirements.txt:35]: do we still need this?

Comment (eddddddy) [reply doc/requirements.txt:35]: Nope.

**Review by rmoyard — APPROVED** [review]
Thanks !

**Review by timmysilv — APPROVED** [review]
woo! unblocking things!

**Pull request merged**